docs/gallery/domain/ecephys.py

```diff
@@ -127,6 +127,19 @@
                             description="Random numbers generated with numpy.random.rand")
 nwbfile.add_acquisition(ephys_ts)
 
+####################
+# .. _units_electrode:
+#
+# Associate electrodes with units
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# The :ref:`PyNWB Basics tutorial <basics>` demonstrates how to add data about units and specifying custom metadata
+# about units. As mentioned :ref:`here <units_fields_ref>`, there are some optional fields for units, one of these
+# is *electrodes*. This field takes the *id* for the electrode that the unit corresponds to. For example, if two units
+# were inferred from the first electrode (*id* = 0), you would be specify that like so:
+
+nwbfile.add_unit(id=1, electrode=0)
+nwbfile.add_unit(id=2, electrode=0)
```

Comment:
this syntax is nice. Would it be the same for `electrodes`? And would you mind adding some tests as well?

Reply:
I guess we throw when the electrode with the given id does not exist? Or maybe accepting a electrode object would be better?

Reply:
@bendichter There is no electrodes field in the units table. Right now, you can only specify a one-to-one relationship. I’ll fix the typo in the doc. 

@t-b as is, the check will happen at write.